trpcultivate_phenoshare/src/Plugin/TripalImporter/TripalCultivatePhenoshareImporter.php

```diff
@@ -1,44 +1,38 @@
 <?php
 
-/**
- * @file
- * Tripal Importer Plugin implementation for Tripal Cultivate Phenotypes - Share.
- */
-
 namespace Drupal\trpcultivate_phenoshare\Plugin\TripalImporter;
 
 use Drupal\tripal_chado\TripalImporter\ChadoImporterBase;
-use Drupal\Core\Url;
 use Drupal\tripal_chado\Controller\ChadoProjectAutocompleteController;
-use Drupal\tripal_chado\Database\ChadoConnection;
 use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Drupal\Core\Ajax\AjaxResponse;
 use Drupal\Core\Ajax\InvokeCommand;
-use Drupal\Core\Ajax\ReplaceCommand;
+use Drupal\tripal_chado\Database\ChadoConnection;
+use Drupal\trpcultivate_phenotypes\Service\TripalCultivatePhenotypesGenusOntologyService;
 
 /**
  * Tripal Cultivate Phenotypes - Share Importer.
  *
- * Focused on phenotypic data which has already been published or which is ready
- * to be freely shared.
+ * An importer focused on phenotypic data which has already been published or
+ * which is ready to be freely shared.
  *
  * @TripalImporter(
  *   id = "trpcultivate-phenotypes-share",
- *   label = @Translation("Tripal Cultivate: Open Science Phenotypic Data"),
+ *   label = @Translation("Tripal Cultivate: Phenotypic Share Importer"),
```

Comment:
I don't agree with this change. The importer was named the "Open Science Phenotypic Data" importer since this will be forward facing to the researchers. Adding the "Share" back in the name feels more developer-speak to me personally 🤷‍♀️